### DISCUSSION TITLE
易控车机版讨论区

### DISCUSSION THREAD

**Discussion by eiyooooo** [eiyooooo/Easycontrol_For_Car#8] [General]
有任何问题或者不明白的地方可以在这里提出哦👋

**Comment by duoxini** [comment]
![IMG_20240416_122933.jpg](https://github.com/eiyooooo/Easycontrol_For_Car/assets/60737576/0cdcaf6b-2f08-41cd-9b0d-4ce15f78b2ed)

右边高德为手机投屏应用流转，该窗口不能再往上移动了，可能是什么原因？

**Reply by eiyooooo** [reply]
上方检测到状态栏了，如果继续往上移动可能就拖不回来了

**Reply by duoxini** [reply]
好吧，
提个建议：通过坐标方式来调整窗口位置，并提供窗口大小输入参数以进行调整。

**Reply by duoxini** [reply]
另外手机音频破音比较厉害，手机是小米13，安卓14

**Reply by eiyooooo** [reply]
可以的话尽量连蓝牙吧🫡
